llvm/test/compiler-course/beresnev_a/test_llvm_ir.ll

```diff
@@ -0,0 +1,29 @@
+; RUN: opt -load-pass-plugin %llvmshlibdir/UserPass_Beresnev_Anton_FIIT1_LLVM_IR%pluginext\
+; RUN: -passes=pure_fn -S %s | FileCheck %s
+
+; CHECK: Function _Z3addii marked as pure
+; CHECK: Function _Z3foof NOT pure
```

Comment:
The test should check for the attribute (it is observable in LLVM IR) instead of checking for the pass console output